apx/common/src/apx_fileManager.c

```diff
@@ -457,12 +457,20 @@ void apx_fileManager_triggerFileWriteCmdEvent(apx_fileManager_t *self, apx_file_
       }
       else
       {
+         uint8_t fileWriteCmdEventResult;
          memcpy(dataCopy, data, length);
          msg.msgData4 = dataCopy;
          SPINLOCK_ENTER(self->lock);
-         rbfs_insert(&self->ringbuffer,(const uint8_t*) &msg);
+         fileWriteCmdEventResult = rbfs_insert(&self->ringbuffer,(const uint8_t*) &msg);
          SPINLOCK_LEAVE(self->lock);
-         SEMAPHORE_POST(self->semaphore);
+         if (fileWriteCmdEventResult != E_BUF_OK)

```

Comment:
You can actually improve here. When buffer is full it returns E_BUF_OVERFLOW. You should have one specific log warning for when buffer is full and another log warning for other types of errors